### PR TITLE
Added and fixed Polish phrases

### PR DESCRIPTION
**Pull request by strebski**
Some autocorrection from previous update

### CODE REVIEW

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -3359,27 +3384,34 @@
...
     en = boundary-administrative-city
+    pl = Granica miasta
 
   [type.boundary.administrative.country]
     en = boundary-administrative-country
+    pl = Granica państwa
 
   [type.boundary.administrative.county]
     en = boundary-administrative-county
+    pl = Granica państwa
```
Comment (mpimenov) [inline data/strings/types_strings.txt]: Are you sure?
https://wiki.openstreetmap.org/wiki/Tag:place%3Dcounty

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -11093,19 +11153,23 @@
...
     en = landuse-retail
     pl = Obszar handlu
 
   [type.landuse.salt_pond]
     en = landuse-salt_pond
+    pl = Słone jezioro
 
   [type.landuse.village_green]
     en = landuse-village_green
+    pl = Nawsie
```
Comment (mpimenov) [inline data/strings/types_strings.txt]: are you sure?
https://pl.wikipedia.org/wiki/Nawsie

File: data/strings/types_strings.txt (hunk 1 above)
Comment (strebski) [reply data/strings/types_strings.txt]: You're right. Should be "pl = Hrabstwo", like it is described on Polish version of mentioned wiki site. Thanks for control!

File: data/strings/types_strings.txt (hunk 2 above)
Comment (strebski) [reply data/strings/types_strings.txt]: Yes, in this issue "Nawsie" is correct. Another meaning of this word: https://pl.m.wikipedia.org/wiki/Nawsie_(urbanistyka)
Also check
https://wiki.openstreetmap.org/wiki/Pl:Tag:landuse%3Dvillage_green

File: data/strings/types_strings.txt (hunk 1 above)
Comment (strebski) [reply data/strings/types_strings.txt]: "pl = Granica hrabstwa", of course.

File: data/strings/types_strings.txt (hunk 2 above)
Comment (mpimenov) [reply data/strings/types_strings.txt]: Looks like a niche term but ok.

Comment (strebski) [reply data/strings/types_strings.txt]: "Nawsie" is not popular Polish word, you are right :-)

File: data/strings/types_strings.txt (hunk 1 above)
Comment (strebski) [reply data/strings/types_strings.txt]: County boundary already fixed!

**Pull request merged**